build/scripts/deploy-msbuild.ps1

```diff
@@ -0,0 +1,51 @@
+
+# This script will ensure that the MSBuild toolset of the specified 
+# version is downloaded and deployed to Binaries\Toolset\MSBuild
+#
+# This is a no-op if it is already present at the appropriate version
+
+param ()
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+$version = "0.1.7-alpha"
```

Comment:
nit: Make version a parameter with a default value of "0.1.7-alpha", it makes it easier to test new changes